tests/testutils.py

```diff
@@ -32,37 +32,8 @@
 from testconfig import dsn, repl_dsn
 
 
-if hasattr(unittest, 'skipIf'):
-    skip = unittest.skip
-    skipIf = unittest.skipIf
-
-else:
-    import warnings
-
-    def skipIf(cond, msg):
-        def skipIf_(f):
-            @wraps(f)
-            def skipIf__(self):
-                if cond:
-                    with warnings.catch_warnings():
-                        warnings.simplefilter('always', UserWarning)
-                        warnings.warn(msg)
-                    return
-                else:
-                    return f(self)
-            return skipIf__
-        return skipIf_
-
-    def skip(msg):
-        return skipIf(True, msg)
-
-    def skipTest(self, msg):
-        with warnings.catch_warnings():
-            warnings.simplefilter('always', UserWarning)
-            warnings.warn(msg)
-        return
-
-    unittest.TestCase.skipTest = skipTest
+skip = unittest.skip
+skipIf = unittest.skipIf
```

Comment:
I believe `skip` and `skipIf` are unused. Might as well remove them.

Reply:
If they are unused, yes